docs/errors/validation_errors.md

```diff
@@ -778,8 +778,7 @@ except ValidationError as exc:
 
 ## `int_parsing_size`
 
-This error is raised when attempting to parse a python value from a string outside the maximum range of a Python `int`,
-or when trying to parse an integer from JSON that is outside the valid range of a 64 bit integer:
+This error is raised when attempting to parse a python value from a string outside the maximum range of a Python `int`:
```

Comment:
We still get this error message in JSON parsing, So the second line shouldn't be deleted. You need to adjust the second line I think